### ISSUE TITLE
Referencing MSForms

### ISSUE THREAD

**Issue by bclothier** [WaynePhillipsEA/twinbasic#89]
**Describe the bug**
If you try to add a reference to the MS Forms, 2.0, you get several possible choices:
![image](https://user-images.githubusercontent.com/2367644/115863372-fa371f00-a3fa-11eb-92ff-6f2323538566.png)

This was motivated by the fact that normally, I do not have MS Forms 2.0 library available in my Windows directory. On this machine, I only have Office 365 installed. FM20.DLL is virtualized and is found in this path (assuming 64-bit Office):

```
C:\Program Files\Microsoft Office\root\vfs\System\FM20.DLL
```

However, I think in this case, we have multiple instances on the account of Office copying the FM20.DLL into a new library everytime it is used. 

Furthermore, I find that the library does not seem to be complete. I can:

```
Dim c As MSForms.Control
````

But not:

```
Dim c As MSForms.TextBox
```

Several classes seems to be missing:
![image](https://user-images.githubusercontent.com/2367644/115864110-0ec7e700-a3fc-11eb-8d00-c1e0cc913d27.png)

**To Reproduce**
Note: this may be influenced b y Office installs. This was from a machine that only has Office 365 installed. FM20.DLL does not exist in `C:\Windows\System32\` as it would have in older version of Office.:

1. Create a twinBASIC project. Shouldn't matter which project.
2. Add a reference to MS Forms 2.0 library. You may find several.
3. Try to define `Dim x As MSForms.` one of the common controls or class such as `UserForm` or `TextBox`. Those will not be available. 

**Expected behavior**
twinBASIC may need to handle Office365's virtualizing of the DLLs and the registry entries in order to locate the library that are otherwise sandboxed in Office365's environment. There should be only one `MSForms 2.0` reference and that it should contian all the objects.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - twinBASIC compiler version 0.9.1944

**Comment by WaynePhillipsEA** [comment]
Thanks @bclothier, will have a look at this over the weekend to see what's going on.

**Comment by retailcoder** [comment]
I did notice the library appeared to be split up, but didn't think much of it and just referenced all 3, and it started compiling; I don't think they're (all?) duplicates.

**Comment by bclothier** [comment]
I did not check every one but I did check first and last and maybe 2nd and they did not seem to be different and all were missing expected members.

**Comment by bclothier** [comment]
Just as a follow up - I tried again and referenced them all. I did not find any controls one usually expect in the library.

**Comment by WaynePhillipsEA** [comment]
This should now be fixed in 0.10.3988.

Turns out that when VBA adds a reference to the Microsoft Forms 2.0 Object Library (i.e. after adding your first UserForm to a project), it creates a special customized version of this type library with the EXD file extension, and registers it in the registry.  Each time with a different typelib GUID, and pointing to a file in the temp folder.  So you can end up with masses of these entries if you use UserForms a lot.   These special customized versions have the LIBFLAG_FHIDDEN bit set in the FLAGS value in the registry, so we can easily filter them out, which is what happens now as of v0.10.3988.
